tests/v2/validation/provisioning/k3s/README.md

```diff
@@ -22,13 +22,17 @@ provisioningInput is needed to the run the K3S tests, specifically kubernetesVer
 
 ```json
 "provisioningInput": {
+    // for custom clusters, len(nodesAndRoles) should be equal to len(awsEc2Config)
     "nodesAndRoles": [
       {
         "etcd": true,
         "controlplane": true,
         "worker": true,
         "quantity": 1,
       },
+      {
+        "quantity": 2,
```

Comment:
RKE1 and RKE2 show `worker` defined along with `quantity`. Should that be the case in the K3S README as well?